src/fragments/lib/graphqlapi/ios/subscribe-data.mdx

```diff
@@ -60,7 +65,12 @@ func createSubscription() {
 ```swift
 func createSubscription() {
     subscription = Amplify.API.subscribe(request: .subscription(of: Todo.self, type: .onCreate))
-    dataSink = subscription?.subscriptionDataPublisher.sink {
+    guard let subscription = subscription else {
+        print("Error getting subscription")
+        return
+    }
+    dataSink = Amplify.Publisher.create(subscription)
```

Comment:
same here, `let sequence = Amplify.API.subscribe..`

Reply:
with the above, you can then keep the name of the AnyCancellable the same as before, `var subscription: AnyCancellable?`